gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageOptions.java

```diff
@@ -198,6 +199,8 @@ public static Builder builder() {
 
   public abstract MetricsSink getMetricsSink();
 
+  public abstract Boolean getTraceLogEnable();
```

Comment:
Use `Enabled` in the getter and setter method names (see other boolean configs)

Reply:
Done